src/main/java/com/assignment/Assignment2.java

```diff
@@ -0,0 +1,14 @@
+package com.assignment;
+
+public class Assignment2 {
+    public static String getReplacedCharacter(String originalString,char characterToBeReplaced, char replacement){
+        return originalString.replace(characterToBeReplaced,replacement);
+    }
+    public static String getReplacedSubString(String originalString,String subStringToBeReplaced, String replacement){
+        return originalString.replace(subStringToBeReplaced,replacement);
+    }
```

Comment:
Why are we having 2 methods, when the code inside the method is doing the same thing?